dev-infrastructure/docs/development-setup.md

```diff
@@ -122,6 +122,12 @@ This will create:
 1. A custom role definition with required access as defined in `dev-infrastructure/scripts/mock-dev-role-definition.json`
 1. A service principal/application using the created cert as its authentication, and given access based on the custom role definition
 
+In the newly created Resource Group (test-RG):
```

Comment:
Not sure why this is in the `Step 2 - Create the Application and its dependencies` section? It's not related. This is a pre-requisite of creating an ARO HCP cluster, I would expect this to be closer to that documentation.